src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/ApprovalController.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.companieshouse.web.accounts.controller.smallfull;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping(value = "/company/{companyNumber}/transaction/{transactionId}/company-accounts/{companyAccountsId}/small-full/approval")
```

Comment:
No need to explicitly specify `value =`. If you provide a String to a `@RequestMapping` it sets the `value` attribute by default.